frontend/src/store/api-store.spec.js

```diff
@@ -1,5 +1,5 @@
 import { createLocalVue, mount } from '@vue/test-utils'
```

Comment:
Auch diese Tests (und das dazugehörige .eslintrc.js und die resources) würde ich in einen `__tests__` Unterordner verschieben

Reply:
Mich stören diese Underlines extrem. Kann es nicht einfach `test` (oder `tests`) heissen?

Reply:
Es ist Konvention in Jest: https://github.com/facebook/jest/issues/637
Aber wir können es von mir aus gerne anpassen, musst einfach in package.json die `"testMatch"` auch ändern.
@usu wie siehst du das?

Reply:
Okay, ich sehe vuetify macht auch `__tests__`
Ich gebe mich geschlagen ;)